### PR TITLE
Windows Server 2022 Update to Bitlocker PowerShell Module 

### PR DESCRIPTION
**Pull request by IngridAtMicrosoft**

### CODE REVIEW

**Comment by IngridAtMicrosoft** [comment]
#sign-off

File: docset/winserver2022-ps/bitlocker/BitLocker.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ This reference provides cmdlet descriptions and syntax for all BitLocker cmdlets
 ### [Add-BitLockerKeyProtector](./Add-BitLockerKeyProtector.md)
 Adds a key protector for a BitLocker volume.
 
+### [BackupToAAD-BitLockerKeyProtector](.BackupToAAD-BitLockerKeyProtector.md)
+Saves a key protectore for a BitLocker volume in AAD.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/bitlocker/BitLocker.md:21]:
```suggestion
Saves a key protector for a BitLocker volume in AAD.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @IngridAtMicrosoft, kindly commit the suggested change to fix the typo. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by IngridAtMicrosoft** [comment]
@traya1  This is ready to be merged. Thanks!

Ingrid

File: docset/winserver2022-ps/bitlocker/BackupToAAD-BitLockerKeyProtector.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+## DESCRIPTION
+The **BackupToAAD-BitLockerKeyProtector** cmdlet saves a recovery password key protector for a volume protected by BitLocker Drive Encryption to Azure Active Directory (AAD). Specify a key to be saved by ID.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+$BLV = Get-BitLockerVolume -MountPoint "C:"
+BackupToAAD-BitLockerKeyProtector -MountPoint "C:" -KeyProtectorId $BLV.KeyProtector[1].KeyProtectorId
```
Comment (dariomws) [inline docset/winserver2022-ps/bitlocker/BackupToAAD-BitLockerKeyProtector.md:28]:
```suggestion
PS C:\> $BLV = Get-BitLockerVolume -MountPoint "C:"
PS C:\> BackupToAAD-BitLockerKeyProtector -MountPoint "C:" -KeyProtectorId $BLV.KeyProtector[1].KeyProtectorId
```

File: docset/winserver2022-ps/bitlocker/BackupToAAD-BitLockerKeyProtector.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+This example saves a key protector for a specified BitLocker volume.
+
+The first command uses **Get-BitLockerVolume** to obtain a BitLocker volume and store it in the $BLV variable.
+
+The second command backs up the key protector for the BitLocker volume specified by the MountPoint parameter. The command specifies the key protector by using its ID, contained in the BitLocker object stored in $BLV. 
+
+### Example 2
+```powershell
+BackupToAAD-BitLockerKeyProtector -MountPoint "C:" -KeyProtectorId "{E2611001E-6AD0-4A08-BAAA-C9c031DB2AA6}"
```
Comment (dariomws) [inline docset/winserver2022-ps/bitlocker/BackupToAAD-BitLockerKeyProtector.md:40]:
```suggestion
PS C:\> BackupToAAD-BitLockerKeyProtector -MountPoint "C:" -KeyProtectorId "{E2611001E-6AD0-4A08-BAAA-C9c031DB2AA6}"
```

File: docset/winserver2022-ps/bitlocker/BackupToAAD-BitLockerKeyProtector.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/bitlocker/BackupToAAD-BitLockerKeyProtector.md:109]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/bitlocker/BitLocker.md [hunk 5]
```diff
@@ -17,6 +17,9 @@ This reference provides cmdlet descriptions and syntax for all BitLocker cmdlets
 ### [Add-BitLockerKeyProtector](./Add-BitLockerKeyProtector.md)
 Adds a key protector for a BitLocker volume.
 
+### [BackupToAAD-BitLockerKeyProtector](.BackupToAAD-BitLockerKeyProtector.md)
```
Comment (dariomws) [inline docset/winserver2022-ps/bitlocker/BitLocker.md:20]:
```suggestion
### [BackupToAAD-BitLockerKeyProtector](./BackupToAAD-BitLockerKeyProtector.md)
```

**Review by dariomws — CHANGES_REQUESTED** [review]
Minor suggestions.

**Review by get-itips — APPROVED** [review]
I am ok once Dario's suggestions are applied

**Review by tseward — APPROVED** [review]

**Pull request merged**